src/core/transport/Transport.ts

```diff
@@ -221,7 +220,7 @@ export abstract class AbstractTransport
    * It is one technique for compressing JSON data.
    */
   private handleCborMessage(cbor: ArrayBuffer) {
-    const data: unknown = CBOR.decode(cbor, typedArrayTagger);
+    const data: unknown = decode(new Uint8Array(cbor));
```

Comment:
Uint8Arrays are views on ArrayBuffers, so this (thankfully) doesn't do any copying of potentially large data.